### PR TITLE
Replace Validation error with warning messages

### PR DESCRIPTION
**Pull request by Jason0214**
Now having a not perfect blend file would not raise ValidationError,  it would throw warning messages but still complete the exporting.  It would be much more user friendly and solve some inconvenience mentioned in #78  and #69

### CODE REVIEW

**Comment by sdfgeoff** [comment]
If I recall correctly, there's already a bunch of stuff written to python's logging system. In my mind it would make more sense to use that rather than a custom "message queue"

I like what this PR does, but I'd prefer to see it implemented as a custom logging handler.

**Comment by Jason0214** [comment]
the problem of python logging is that it can not prompt into GUI, while this one does

**Comment by Jason0214** [comment]
Sorry, I didn't get your idea 😅, rewrite it with python logging, it works perfect.

File: io_scene_godot/converters/animation.py [hunk 1]
```diff
@@ -472,37 +469,45 @@ def to_matrix(self):
         # are aggregated to object while being evaluted
         object_path, attribute = split_fcurve_data_path(fcurve.data_path)
 
-        if object_path not in transform_frame_values_map:
```
Comment (Jason0214) [inline io_scene_godot/converters/animation.py]: Here I combine two if branches into one to avoid pylint warning

**Pull request merged**